src/cpp/session/modules/SessionCodeTools.R

```diff
@@ -2587,3 +2587,31 @@
    getNamespace(envir)
    
 })
+
+.rs.addJsonRpcHandler("project_guess_cpp_style", function(inDirectory = .rs.getProjectDirectory()) 
+{
+   # give up if this is not a project or not a package
+   if (is.null(inDirectory)) return(.rs.scalar(""))
```

Comment:
```suggestion
   if (is.null(inDirectory))
      return(.rs.scalar(""))
```

nit: prefer return statements on their own blocks